motion/p5js/cones/index.md

```diff
@@ -3,7 +3,7 @@ layout: p5js
 author: Natalie Schreiber
 author-url: https://github.com/natascher
 title: cones
-tags: [P5.js, motion, "Law of Simplicity"]
+tags: [P5.js, motion, Law of Space through Light and Shadow]
```

Comment:
Currently there is only the tag for the "Law of Space" for the sake of the current state please change it to only that.